src/physics/em/detail/RelativisticBremDXsection.i.hh

```diff
@@ -26,12 +27,11 @@ CELER_FUNCTION RelativisticBremDXsection::RelativisticBremDXsection(
     , elem_data_(shared.elem_data[material.element_id(elcomp_id)])
     , total_energy_(particle.energy().value() + shared.electron_mass.value())
 {
-    real_type density_factor = material.electron_density()
-                               * this->migdal_constant();
-    density_corr_ = density_factor * ipow<2>(total_energy_);
-
-    lpm_energy_ = material.radiation_length() * this->lpm_constant();
+    real_type density_factor = material.electron_density() * migdal_constant();
+    density_corr_            = density_factor * ipow<2>(total_energy_);
 
+    lpm_energy_ = material.radiation_length()
+                  * (lpm_constant() / native_value_from(units::MevEnergy{1}));
```

Comment:
See above units comment: this should just be:
```c++
lpm_energy_ = material.radiation_length() * value_as<MevPerCm>(lpm_constant());
```
so that `lpm_energy_` is in units of MeV which is I think what you want

Reply:
And if `lpm_constant()` really is in native units, you should write this statement as
```c++
    lpm_energy_ = native_value_to<MevEnergy>(material.radiation_length() * lpm_constant()).value();
```

Reply:
Okay, changed to value_as<MevPerCm>(lpm_constant()).